### DISCUSSION TITLE
MPRIS support please

### DISCUSSION THREAD

**Discussion by dimitrikarpov** [Rafostar/clapper#50] [Ideas]
Wow. This software is really awesome. It would be great if Clapper can be controlled using standard media keys.

**Comment by Rafostar** [comment]
Hi, MPRIS will definitely be added. My aim for first release was to just to finish GUI and get it to work.

**Reply by Rafostar** [reply]
Web application interface also needs to be expanded btw. Right now the only thing you can do from web browser is to toggle play/pause.

**Comment by Rafostar** [comment]
MPRIS is now available on latest git :-)
